src/components/Organisms/MarketingPreferencesDS/_MarketingPreferencesDS.js

```diff
@@ -61,13 +61,23 @@ const MarketingPreferencesDS = ({
 
   const customId = id ? `marketing-preferences--${id}` : 'marketing-preferences';
 
+  // Check for field-specific errors
+  const hasEmailError = Boolean(errors.mp_permissionEmail || errors.mp_email);
```

Comment:
Do we actually see errors on `mp_permissionEmail` et al? Aren't these just optional checkboxes? Not sure where an error would happen there?